opm/parser/eclipse/EclipseState/Schedule/DynamicState.hpp

```diff
@@ -152,6 +153,14 @@ class DynamicState {
         return std::distance( m_data.begin() , iter );
     }
 
+    template<typename P>
+    int find_if(P&& pred) const {
+        auto iter = std::find_if(m_data.begin(), m_data.end(), pred);
```

Comment:
This should probably be phrased as
```C++
auto iter = std::find_if(m_data.begin(), m_data.end(), std::forward<P>(pred));
```
We need `forward<P>` (from header `<utility>`) to preserve the value category of `pred`.

Reply:
OK - I got to read about that; ;-)